src/Parser.php

```diff
@@ -438,16 +419,43 @@ protected function parseOutline()
             throw new UnexpectedParserNodeException('Step or Examples table', $node, $this->file);
         }
 
-        if (count($examples) === 0) {
-            throw new ParserException(sprintf(
-                'Outline should have examples table, but got none for outline "%s" on line: %d%s',
-                rtrim($title),
-                $line,
-                $this->file ? ' in file: ' . $this->file : ''
-            ));
+        if ($examples !== []) {
+            return new OutlineNode(rtrim($title) ?: null, $tags, $steps, $examples, $keyword, $line);
         }
 
-        return new OutlineNode(rtrim($title) ?: null, $tags, $steps, $examples, $keyword, $line);
+        return new ScenarioNode(rtrim($title) ?: null, $tags, $steps, $keyword, $line);
```

Comment:
Note that at the moment calling `->getKeyword()` on the parsed node returns a hardcoded result. So if the feature file used `Scenario:` but had an examples table then e.g. Behat's pretty printer would still print `Scenario Outline:`. 

We could add a new property into the nodes to record the actual keyword that we found, but I would be a little nervous about BC impact if the result of `OutlineNode->getKeyword()` can vary in case anyone is matching on / doing anything specific with that string.